domains/codedit.is-an.app.js

```diff
@@ -0,0 +1,12 @@
+addSubDomain({
+  description: 'CodedIt WebContainers',
+  domain: 'is-an.app', 
+  subdomain: 'codedit',
+  owner: {
+    repo: 'https://github.com/SX-9/codedit-web',
+    email: 'sx-91@outlook.com',
+  },
+  record: {
```

Comment:
No `CNAME` or `A` record - is that ok?

Reply:
https://github.com/tarampampam/free-domains/blob/cf1f90b50133f6abb8e113c968d5f194953d7b32/README.md?plain=1#L125-L126

Reply:
its a vercel record so yes

Reply:
![SmartSelect_20230328-213550_Chrome](https://user-images.githubusercontent.com/90962949/228273487-d6f3b97d-207e-4a57-97de-1cfd0e03fa9f.jpg)

Reply:
So, replace the txt record with cname, please. Since txt is needed only to verify the domain - I will append it manually

Reply:
alr

Reply:
🤦

Reply:
https://github.com/tarampampam/free-domains/blob/cf1f90b50133f6abb8e113c968d5f194953d7b32/domains/yt2mp3.is-an.app.js#L10

https://github.com/tarampampam/free-domains/issues/97

Reply:
Ping @SX-9

Reply:
OOOOOH ok ok sry

Reply:
now is it good?